project-test/docker-compose.yml

```diff
@@ -20,7 +20,8 @@ services:
             XDEBUG_CONFIG: '${SAIL_XDEBUG_CONFIG:-client_host=host.docker.internal}'
         volumes:
             - '.:/var/www/html'
-            - '../package:/var/www/package'
+            - '../:/var/www/package'
+            - /var/www/package/project-test
```

Comment:
Line 24 doesn't look like a volume mount :thinking:

Reply:
I found this tip to exclude folders. I checked this path, and the folder is empty :grin:

Reply:
ah, nice!